bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartGraphs.java

```diff
@@ -18,6 +18,7 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.LinkedHashMap;
+import java.util.Map;
 
```

Comment:
I discovered that showing the slideout with custom data is very slow, so I profiled it and it takes almost 1 second to layout 52 toolbars. 

I use this profiling tool for MT since 10+ years to find bottlenecks when they are discovered.

![mt-profiling](https://user-images.githubusercontent.com/1283445/105992582-9a540500-60a5-11eb-82c3-32765b408669.png)

Reply:
I'm not happy with the slideout UI, it's just lake a maze to find a graph, there is no sorting or categorzation. 

I would use a column format
* 1st column contains all already existing graphs
* 2nd .. nth column contains the custom track graph

For this screenshot, I removed the inner border and resized the slideout to show ALL graphs

![mt-all-chart-graphs](https://user-images.githubusercontent.com/1283445/105993304-865cd300-60a6-11eb-8ed8-929ca5eeb793.png)


The graph min/max slideout is already showing the existing graphs in a column and the displaying time is much faster compared with the graph slideout with custom tracks

![mt-graph-min-max](https://user-images.githubusercontent.com/1283445/105993662-f53a2c00-60a6-11eb-87e0-c7efe6979b92.png)

Reply:
The columns are also more or less randomly sorted, not very user friendly

![mt-cust-track-columns](https://user-images.githubusercontent.com/1283445/105994016-5c57e080-60a7-11eb-8223-71f1e114744d.png)

Reply:
@wolfgang-ch  thanks. Does your profiler gives a hint on why the code is slow, meaning what could be done to make it faster ?
1) I'll modify the display to be in Alphabetical Order
2) I'll investigate how to transform in "Column Format"
3) change 'cnt' to 'count'

Reply:
>Does your profiler gives a hint on why the code is slow, meaning what could be done to make it faster ?

No, however you can see the code location where it occurred

>     3. change 'cnt' to 'count'

`count` for what ?

Since a while I use the prefix `num` (reading: "number of") for fields which are counting something, e.g.

![mt-num](https://user-images.githubusercontent.com/1283445/106096313-98d21d80-6135-11eb-833c-66e2648e540c.png)